lib/std/os/bits/linux/bpf.zig

```diff
@@ -148,6 +200,130 @@ pub const BPF_F_CLONE = 0x200;
 /// flag for BPF_MAP_CREATE command. Enable memory-mapping BPF map
 pub const BPF_F_MMAPABLE = 0x400;
 
+/// These values correspond to "syscalls" within the BPF program's environment
+pub const Helper = enum(i32) {
```

Comment:
Can we give this a more descriptive name? Maybe `Op` ?

Reply:
So "bpf helper" is what they're officially referred to as, and I agree that the name is atrocious -- it took me a while to find that these weren't actually helper functions. I'm ok with renaming and making a note in docs. What do you think of `Call` since they're used in the call instruction?

Reply:
If "Helper" is the official name, then that's fine by me. It's your call if you think it's worth it to break from the established convention in this case.

Reply:
Yeah I couldn't find a different naming convention so I'm keeping "Helper"